pkg/platform/api/platform.go

```diff
@@ -53,7 +53,11 @@ type listAvailableResponse struct {
 }
 
 func (lar *listAvailableResponse) Updates() []platform.Update {
-	return []platform.Update{lar.chosenUpdate}
+	if lar.chosenUpdate != nil {
+		return []platform.Update{lar.chosenUpdate}
+	}
+
+	return []platform.Update{}
```

Comment:
```suggestion
	if lar.chosenUpdate == nil {
		return nil
	}

	return []platform.Update{lar.chosenUpdate}
```

This would also work (unless there's some reason not to return `nil` from this function.

But please do not block on this suggestion!

Reply:
This is what I originally had. I removed it during my PR by mistake at some point. Sorry for the trouble :(
https://github.com/bottlerocket-os/bottlerocket-update-operator/blob/ccf3431090e036828bf511162d9e0b7cd927df60/pkg/platform/api/platform.go#L51-L54

Reply:
Yeah, there's no call sites that need the array to be hydrated for use. Updated to use `return nil`.